mainnet_contract/contracts/LiteCheckpoint.sol

```diff
@@ -69,7 +71,7 @@ contract LiteCheckpoint {
      * 1. Verify subnet header meta information
      * 2. Verify subnet header certificates
      * 3. (Conditional) Update Committed Status for ancestor blocks
-     * 4. header0 always is gap/epoch and next headers is commit header0
+     * 4. header0 always gap/epoch and next headers is commit header0
```

Comment:
remove another "is"

Reply:
this one have removed , pls git pull again